Update rplidar.md with instructions for finding device path on mac OS… (#1665)
viamrobotics/docs: docs/extend/modular-resources/examples/rplidar.md

![adding rplidar component](/extend/modular-resources/add-rplidar/add-rplidar-component-ui.png)

Paste the following into the **Attributes** field of your new component according to your machine's architecture (none needed for Linux).

> 
> If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration.
> For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion

If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration. 
For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:

```

{{% tab name="JSON Template" %}}

Navigate to the **Config** tab.

> Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON. For M1 & M2, you will need to check your system's output for `ls /dev/ | grep tty.usbserial` and put the number seen there. For example, you may see `tty.usbserial-130`, in which case you should replace XXX with 130:

Reviewer: ```suggestion
Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON.
If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration. 
For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:
```

{{% tab name="JSON Template" %}}

Navigate to the **Config** tab.

> Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON.
> If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration. 
> For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:

Reviewer: ```suggestion
Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON:
```
